### DISCUSSION TITLE
Windows 32bit -- version 0.2

### DISCUSSION THREAD

**Discussion by MRIIOT** [Ladder99/fanuc-driver#56] [Announcements]
[Watch Windows installation procedure video.](https://youtu.be/_sZ0qCVkTkM)  
[Whats new in version 0.2.]()

Notes:

1. Built against [aad808a](https://github.com/Ladder99/fanuc-driver/commit/aad808a556462c694719fd6dc894062457dee760).

2. [.NET Core 3.1 Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-3.1.22-windows-x86-installer) must be installed prior to running fanuc.exe.
3. [Exclude](https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26) fanuc.exe process or containing folder from antivirus real-time scan for improved system performance.
4. Linux Docker images available [here](https://hub.docker.com/r/ladder99/fanuc-driver).

<hr /><em>This discussion was created from the release <a href='https://github.com/Ladder99/fanuc-driver/releases/tag/0.2'>Windows 32bit -- version 0.2</a>.</em>

**Comment by capxax** [comment]
that is nice job
thx a lot
